src/System.Private.CoreLib/shared/Internal/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -387,5 +387,42 @@ public static IntPtr ByteOffset<T>(ref T origin, ref T target)
         {
             throw new PlatformNotSupportedException();
         }
+
+        /// <summary>
+        /// Returns a by-ref that is a null reference to a type <typeparamref name="T"/>.
+        /// Note this is the address of the by-ref not what the ref points to.
```

Comment:
Nit: "…to type T" (no article needed).
Actually you could use the same wording as for `IsNullRef<T>`: "Returns a by-ref to type T that is a null reference."
I cannot say I like the "address of the by-ref" part either.  I would rephrase or delete the second line.  Thanks!

Reply:
Changed